components/kyma-environment-broker/internal/component_input_helpers.go

```diff
@@ -115,11 +116,59 @@ func GetClusterIDWithKubeconfig(kubeconfig string) ClusterIDGetter {
 		}
 		cm, err := cs.CoreV1().ConfigMaps("kyma-system").Get(context.Background(), "cluster-info", metav1.GetOptions{})
 		if k8serrors.IsNotFound(err) {
-			return "", nil
+			cm = &v1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
```

Comment:
I think we should store it KEB db. Because it if will be removed from cluster we will need to contact service manager guys to get that ID to keep service instances recovery feature from btp-operator working as expected.